lib/examples/axum/src/main.rs

```diff
@@ -20,17 +20,17 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
 
 	db.use_ns("namespace").use_db("database").await?;
 
-	let app = Router::new()
+	let router = Router::new()
 		.route("/person/:id", post(person::create))
 		.route("/person/:id", get(person::read))
 		.route("/person/:id", put(person::update))
 		.route("/person/:id", delete(person::delete))
 		.route("/people", get(person::list))
 		.with_state(db);
 
-	let addr = SocketAddr::from(([127, 0, 0, 1], 8080));
+	let listener = TcpListener::bind("localhost:8080").await?;
```

Comment:
Is this actually the same? Is there any difference between `localhost` and `127.0.0.1` on all operating systems and all environments?

Reply:
On a machine with IPv6 enabled, `localhost` typically points to both `127.0.0.1` and `::1`. On IPv4 only machines it typically points to `127.0.0.1`. `localhost` is reserved for loopback use so for most, if not all systems they should mostly be the same. Technically `localhost` can point to any address within the `127.0.0.0/8` range but `127.0.0.1` is standard. In any case, `localhost` should work regardless of which loopback address it points to.